src/screens/Assistant/ResultItem.tsx

```diff
@@ -211,24 +274,10 @@ const FootLeg = ({leg, nextLeg}: {leg: Leg; nextLeg?: Leg}) => {
       );
 
   return (
-    <View style={styles.legContainer}>
-      <ThemeText
-        type="label"
-        color="faded"
-        style={[styles.textDeprioritized, styles.time]}
-      >
-        {formatToClockOrRelativeMinutes(leg.expectedStartTime)}
-      </ThemeText>
+    <View style={styles.legContainer} accessibilityLabel={text}>
       <View style={styles.iconContainer}>
```

Comment:
Is this needed?

Reply:
If you're thinking of the extra outer view: Can't see that it is, probably just the result of incomplete cleaning

Reply:
Yeah, either or. Just grouped styles together I think?

Reply:
Yep, glad you pointed that out, had a label before as well. Unnecessary with the nesting here now